modules/mobile/spec/request/appointments_list_request_spec.rb

```diff
@@ -28,6 +28,161 @@
 
     after { Timecop.return }
 
+    describe 'start and end date' do
+      let(:beginning_of_last_year) { (DateTime.now.utc.beginning_of_year - 1.year) }
+      let(:one_year_from_now) { (DateTime.now.utc.beginning_of_day + 1.year) }
+
+      context 'when omitted from query params' do
+        let(:params) { { page: { number: 1, size: 10 }, useCache: false } }
+        let(:request_appointments) do
+          VCR.use_cassette('appointments/get_facilities', match_requests_on: %i[method uri]) do
```

Comment:
FYI, you don't need to do the VCR setup here because the code will stop executing at the watched method and never get to the VCR. Also, we're not testing anything in the response. You can confirm this by adding a test for response.status == 200. It will fail. If you add `.and_call_original` to the end of your `expect_any_instance_of` lines, then you'd get a status of 200 because the rest of your code will execute and use the VCRs.

Reply:
Thanks for catching that. Force of habit. I've removed the VCRs in those specs.